packages/jupyter/src/providers.tsx

```diff
@@ -27,7 +27,8 @@ function makeThebeOptions(
     binderBadgeUrl,
   );
 
-  const options = thebeFrontmatter ? optionsFromFrontmatter : undefined;
+  const options = optionsOverrideFn(thebeFrontmatter ? optionsFromFrontmatter : undefined);
```

Comment:
a parent theme is able to fully manipulate the options, guarding against undesirable options (e..g connecting to a foreign binder) coming in from the myst site, or just being able to patch certain options appropriately for where there parent theme is used/deployed.